unit-tests/run-unit-tests.py

```diff
@@ -459,10 +459,10 @@ def test_wrapper( test, configuration=None, repetition=1, sns=None ):
                 log.debug_unindent()  # just to make it stand out a little more
                 log.d( f'  Failed; retry #{retry}' )
                 log.debug_indent()
-            if no_reset:
+            if no_reset or not sns:
```

Comment:
what is the meaning of "sns"? could you find a name that would be more clear, please?

Reply:
Not the purpose of this quick fix for improving the CI, but pushed a change :)